keyboards/hnk/hnk65/keyboard.json

```diff
@@ -0,0 +1,217 @@
+            {"matrix": [1, 12], "x": 175, "y": 16, "flags": 4},
+            {"matrix": [1, 13], "x": 185, "y": 16, "flags": 4},
+            {"matrix": [1, 14], "x": 210, "y": 16, "flags": 4},
+            {"matrix": [2, 14], "x": 210, "y": 32, "flags": 4},
+            {"matrix": [2, 13], "x": 196, "y": 32, "flags": 4},
+            {"matrix": [2, 11], "x": 161, "y": 32, "flags": 4},
+            {"matrix": [2, 10], "x": 147, "y": 32, "flags": 4},
+            {"matrix": [2, 9], "x": 133, "y": 32, "flags": 4},
+            {"matrix": [2, 8], "x": 119, "y": 32, "flags": 4},
+            {"matrix": [2, 7], "x": 105, "y": 32, "flags": 4},
+            {"matrix": [2, 6], "x": 91, "y": 32, "flags": 4},
+            {"matrix": [2, 5], "x": 77, "y": 32, "flags": 4},
+            {"matrix": [2, 4], "x": 63, "y": 32, "flags": 4},
+            {"matrix": [2, 3], "x": 49, "y": 32, "flags": 4},
+            {"matrix": [2, 2], "x": 35, "y": 32, "flags": 4},
+            {"matrix": [2, 1], "x": 21, "y": 32, "flags": 4},
+            {"matrix": [2, 0], "x": 0, "y": 32, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 48, "flags": 4},
+            {"matrix": [3, 1], "x": 32, "y": 48, "flags": 4},
+            {"matrix": [3, 2], "x": 46, "y": 48, "flags": 4},
+            {"matrix": [3, 3], "x": 60, "y": 48, "flags": 4},
+            {"matrix": [3, 4], "x": 74, "y": 48, "flags": 4},
+            {"matrix": [3, 5], "x": 88, "y": 48, "flags": 4},
+            {"matrix": [3, 6], "x": 104, "y": 48, "flags": 4},
+            {"matrix": [3, 7], "x": 118, "y": 48, "flags": 4},
+            {"matrix": [3, 8], "x": 132, "y": 48, "flags": 4},
+            {"matrix": [3, 9], "x": 146, "y": 48, "flags": 4},
+            {"matrix": [3, 10], "x": 160, "y": 48, "flags": 4},
+            {"matrix": [3, 11], "x": 173, "y": 48, "flags": 4},
+            {"matrix": [3, 13], "x": 196, "y": 48, "flags": 4},
+            {"matrix": [3, 14], "x": 210, "y": 48, "flags": 4},
+            {"matrix": [4, 14], "x": 210, "y": 64, "flags": 4},
+            {"matrix": [4, 13], "x": 196, "y": 64, "flags": 4},
+            {"matrix": [4, 11], "x": 182, "y": 64, "flags": 4},
+            {"matrix": [4, 10], "x": 168, "y": 64, "flags": 4},
+            {"matrix": [4, 9], "x": 154, "y": 64, "flags": 4},
+            {"matrix": [4, 8], "x": 140, "y": 64, "flags": 4},
+            {"matrix": [4, 5], "x": 105, "y": 64, "flags": 4},
+            {"matrix": [4, 2], "x": 39, "y": 64, "flags": 4},
+            {"matrix": [4, 1], "x": 21, "y": 64, "flags": 4},
+            {"matrix": [4, 0], "x": 4, "y": 64, "flags": 4}
+        ],
+        "max_brightness": 170,
+        "saturation_steps": 22,
+        "sleep": true
+    },
+    "url": "https://hnkbd.ca/",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x484E"
+    },
+    "ws2812": {
+        "pin": "B2"
+    },
+    "community_layouts": ["65_ansi"],
+    "layouts": {
+        "LAYOUT_65_ansi": {
```

Comment:
The purpose of this is to abstract the electrical layout from the physical layout.

The fact that your `keymap.c` doesnt match this logical order, and can mostly be observed to an end user as "random" means this is not correct and the order needs to be updated.

This can be further shown by the `default_65_ansi` community keymap not working as expected.